Gopkg.toml

```diff
@@ -17,20 +17,11 @@ required = [
 # TODO switch to tags before PMM 2.0 release
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  # branch = "PMM-2.0"
+  branch = "PMM-4379-update-api-stubs"
 
 # TODO Remove when v1.1.0 is released.
 # https://github.com/AlekSi/pointer/milestone/1
 [[constraint]]
   name = "github.com/AlekSi/pointer"
   branch = "develop"
-
-# We want all PMM components to use the same versions of key dependecies to avoid nasty suprises.
```

Comment:
We don't want it anymore?

Reply:
https://github.com/percona/pmm/pull/395#discussion_r308625889, last point